WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -138,6 +139,17 @@ - (void)viewDidLoad
     [self configureReplyTextView];
     [self configureSuggestionsTableView];
     [self configureKeyboardGestureRecognizer];
+
+    NSDictionary *views         = @{
+        @"tableView"        : self.tableView,
+        @"postHeader"       : self.postHeaderWrapper,
+        @"replyTextView"    : self.replyTextView
+    };
+    self.verticalConstraint = [NSLayoutConstraint constraintsWithVisualFormat:@"V:|[postHeader][tableView][replyTextView]"
```

Comment:
Is there a reason this isn't part of the `configureViewConstraints` method?

Reply:
I was experimenting with the constraint and forgot to move it after.  It will be relocated with the next push.